### PR TITLE
#454: Catalog for dashboard

### PR DESCRIPTION
**Pull request by dsuren1**
## Description
This PR updates Mapstore submodule with stable version (`2021.02.xx`) enabling catalog for dashboards with necessary configs 

## Issues
 - #454 

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Feature

**What is the current behavior?** (You can also link to an open issue here)
Catalog is not available in dashboard

**What is the new behavior?**
Catalog is now available in dashboard editor allowing user to configure catalog services

**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes
 - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by offtherailz — CHANGES_REQUESTED** [review]
Uff... sorry my fault. Master changed a lot so migration guidelines are not so aligned yet. 
Can you verify if the real war file works? I'm afraid it doesn't because of `publicPath` setting. 
Please referer to this:
- https://mapstore.readthedocs.io/en/2021.02.xx/developer-guide/mapstore-migration-guide/
There are some minor things like `publicPath`, changes in pom files and servlet to set.

File: configs/localConfig.json [hunk 1]
```diff
@@ -25,15 +25,15 @@
...
-        "method": "authkey"
+    "urlPattern": ".*geostore.*",
+    "method": "bearer"
+    }, {
+      "urlPattern": "\\/geoserver\\-test.*",
+      "authkeyParamName": "ms2-authkey",
+      "method": "authkey"
+  }, {
+    "urlPattern": ".*rest/config.*",
+    "method": "bearer"
```
[the remark this answers is not in the document]
Comment (dsuren1) [reply configs/localConfig.json:36]: Mmmm... definitely not intentional. Probably a result of resolving conflicts, I believe

[the remark this answers is not in the document]
Comment (offtherailz) [reply configs/localConfig.json:36]: good, please restore it to avoid unneeded changes in the history

**Comment by dsuren1** [comment]
Hi @offtherailz 
Updated as per migration guidelines. Kindly let me know for any changes. Thanks!

**Review by offtherailz — APPROVED** [review]

**Comment by offtherailz** [comment]
LGTM. I didn't  tested it locally. Going to test in test environment

**Pull request merged**